### PR TITLE
Remove default x and MSFT_ from unit/integration test templates - Fixes #82

### LINKED ISSUE #82: Remove Default x and MSFT_ from Unit/Integration Test templates
The Unit/Integration test templates are all configured to default to MSFT as the organization(?) code of the resource being tested (see https://github.com/PowerShell/DscResources/blob/master/Tests.Template/integration_config_template.ps1#L14).

They also default the module code to 'x' as well (see https://github.com/PowerShell/DscResources/blob/master/Tests.Template/integration_template.ps1#L16).

I think to make them more generic and usable for non-MSFT resources these codes should be removed from the defaults of these files.

The examples however should be left showing MSFT_ and x where appropriate (see https://github.com/PowerShell/DscResources/blob/master/Tests.Template/integration_template.ps1#L17).

### PR DESCRIPTION
**Pull request by PlagueHO**
This fixes issue #82.

It also corrects these issues in Tests.Template/integration_config_template.ps1:
1. removes quotes from ```configuration '<ResourceName>_config' {``` as they shouldn't be there.
2. Uses ```<ResourceFriendlyName>``` instead of ```<FriendlyName>``` in the node resource.
3. Imports the DSC Resource module instead of just a specific resource (to match all existing integration tests).

Thanks

### CODE REVIEW

File: StyleGuidelines.md [hunk 1]
```diff
@@ -231,3 +231,18 @@ function New-EtwEvent
     # Implementation
  }
 ```
+
+Call cmdlets using all named parameters instead of positional parameters
```
Comment (TravisEz13) [inline StyleGuidelines.md]: I agree with this as a guideline, but I don't think this is style

**Comment by TravisEz13** [comment]
LGTM.  @KarolKaczmarek  are you ok with this guidance being in the style guide?  I guess you could call it style, but it as actually enforced by the script analyzer (which is not supposed to enforce style.)

**Comment by PlagueHO** [comment]
Doh! Apologies - This should actually have been in a separate PR as it resolves issue #98. But I issued this PR from my master branch (being lazy). 

Sorry about that.

Re: PSSA - it will show up as a Warning in PSSA, but currently PSSA is not checking warning rules in DSC. I have raised an issue to change this though (it is in DSCResource.Tests).

**Comment by TravisEz13** [comment]
Then, I'm ok with it but will give @KarolKaczmarek  some time to comment.

**Comment by KarolKaczmarek** [comment]
I think it's good to add it here. Even though it's called "style guidelines" the purpose of this document is to capture both best styling and coding practices we recommend when contributing to Resource Kit repos. Perhaps we could rename it to something more appropriate.

**Pull request merged**